### PR TITLE
Fix the React test errors for missing props etc.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- run `make start-karma`
- open the browser link
- open your browser console
- when the tests have completed check that there are only a few errors

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4533/
Test PASSed.

File: jujugui/static/gui/src/app/components/env-switcher/env-switcher.js [hunk 1]
```diff
@@ -159,7 +159,7 @@ YUI.add('env-switcher', function() {
       // XXX j.c.sackett 2015-10-28 When we have the UI for template creation
       // and template selection in the GUI, this code should be updated to not
       // select template based on state server.
-      var srvCb = function(error, servers) {
+      var srvCb = (error, servers) => {
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/env-switcher/env-switcher.js]: Nice

**Comment by anthonydillon** [comment]
Code looks good and a lot less errors in QA :+1:

**Comment by hatched** [comment]
Thanks for these updates! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**